src/app/utils/utils.ts

```diff
@@ -265,12 +268,28 @@ export class Utils {
           }         
         }
 
+        if (output.length > 1) {
+          var resultOutputSplitted = output[0].split(this.COMMA);
+          output.slice(1, output.length).forEach(outputArray => {
+            let tempOutputArray = outputArray.split(this.COMMA)
+            for (let j = 0; j < tempOutputArray.length; j++) {
+              if (resultOutputSplitted[j] === this.EMPTY_STRING_CSV) {
+                resultOutputSplitted[j] = tempOutputArray[j];
+              }
+            }
+          });
```

Comment:
I think the code from L272 - L280 needs to be extracted in that method as well, since this part is only related to this concrete problem.

Reply:
@gmakharat Done!